src/libmongoc/src/mongoc/mongoc-flags.h

```diff
@@ -78,7 +78,7 @@ typedef enum {
  * @MONGOC_QUERY_NONE: No query flags supplied.
  * @MONGOC_QUERY_TAILABLE_CURSOR: Cursor will not be closed when the last
  *    data is retrieved. You can resume this cursor later.
- * @MONGOC_QUERY_SLAVE_OK: Allow query of replica slave.
+ * @MONGOC_QUERY_SECONDARY_OK: Allow query of replica slave.
```

Comment:
```suggestion
 * @MONGOC_QUERY_SECONDARY_OK: Allow query of replica secondary.
```

Reply:
I did a second pass over to catch those instances and fixed them.